x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/ranges/range_editor.tsx

```diff
@@ -47,6 +48,12 @@ const BaseRangeEditor = ({
   const granularityLabel = i18n.translate('xpack.lens.indexPattern.ranges.granularity', {
     defaultMessage: 'Granularity',
   });
+  const granularityLabelDescription = i18n.translate(
+    'xpack.lens.indexPattern.ranges.granularityDescription',
+    {
+      defaultMessage: 'Dynamically build fixed-size intervals based on a numeric field.',
```

Comment:
@gchaps I think this tooltip isn't describing how the "granularity" slider would affect the results, and I'm not sure the words "dynamic"  and "on a numeric field" are adding clarity. If I were to describe what the granularity slider does, it would be "Kibana divides the field into evenly spaced intervals based on the granularity". To turn this into an active sentence, I would be happy with the tooltip saying "Divides the field into evenly spaced intervals"

Reply:
@wylieconlon Good suggestion for "Divides the field into evenly spaced intervals." Requires ending period.